draft-ietf-rats-corim.md

```diff
@@ -239,7 +241,14 @@ namespace under which the value must be understood.
 ### Digest {#sec-common-hash-entry}
 
 A digest represents the value of a hashing operation together with the hash
-algorithm used.
+algorithm used.  The type of the digest algorithm identifier can be either
+`int` or `text`.  When carried as an integer value, it is interpreted according
+to the "Named Information Hash Algorithm Registry" {{-named-info}} registry.
+When it is carried as `text`, there are no requirements with regards to its
+format.  In general, the `int` encoding is RECOMMENDED.  The `text` encoding
+should only to be used when the `digest` type conveys reference value
```

Comment:
grammar seems wrong - maybe something like "should only be used when..."?
upper case Reference Value?